akka-persistence/src/main/scala/akka/persistence/PersistentActor.scala

```diff
@@ -309,6 +310,37 @@ abstract class UntypedPersistentActor extends UntypedActor with Eventsourced wit
 abstract class AbstractPersistentActor extends AbstractActor with PersistentActor with Eventsourced {
 
   /**
```

Comment:
define is a bit weird... `initialRecoveryReceive` would be another option? Sounds more close to what we have elsewhere